src/utils/apub_name.rs

```diff
@@ -2,31 +2,31 @@ use cfg_if::cfg_if;
 use std::str::FromStr;
 
 pub fn create_user_apub_name(name: &str, actor_id: &str) -> String {
-  create_apub_name::<'@'>(name, actor_id)
+  create_apub_name::<'@'>(name, actor_id).unwrap_or_default()
 }
 
 pub fn create_community_apub_name(name: &str, actor_id: &str) -> String {
-  create_apub_name::<'!'>(name, actor_id)
+  create_apub_name::<'!'>(name, actor_id).unwrap_or_default()
 }
 
 fn format_apub_name<const PREFIX: char>(name: &str, instance: &str) -> String {
   format!("{PREFIX}{name}@{instance}")
 }
 
-fn create_apub_name<const PREFIX: char>(name: &str, actor_id: &str) -> String {
+fn create_apub_name<const PREFIX: char>(name: &str, actor_id: &str) -> Option<String> {
```

Comment:
I went with an `Option` instead of a `Result` because the `Err` type differed between the different url libraries.

Reply:
Not having to map error types is where [anyhow](https://docs.rs/anyhow/latest/anyhow/) can become helpful, but yeah its pry not necessary rn because I don't know how leptos errors should work.